javascript/javascript-9.md

```diff
@@ -33,6 +33,7 @@
 - [JavaScript Promises in 10 Minutes](https://www.youtube.com/watch?v=DHvZLI7Db8E)
 - [Understanding Promises in JavaScript](https://teamtreehouse.com/library/understanding-promises-in-javascript)
 - [Async/await reference](https://javascript.info/async-await)
+- [Introduction to Promises](https://beta.observablehq.com/@mbostock/introduction-to-promises)
```

Comment:
Thanks for working on this!  Could you please:
1. use this link in "Demonstration" steps 4 and 5, telling them how to follow along with the tutorial to learn about setting up a Promise, .then and setTimeout.
2. Rename "demonstration" to "guided practice"
3. Replace steps 2 and 3 with actual steps for doing this, rather than expecting the student to make it up.

Reply:
Yes, I will do that